ide/che-core-ide-stacks/src/main/resources/stacks.json

```diff
@@ -1,5 +1,75 @@
 [
   {
+    "id": "java-mysql",
+    "creator": "ide",
+    "name": "Java-MySQL",
+    "description": "Multi-machine environment with Default Java Stack and MySQL database",
+    "scope": "general",
+    "tags": [
+      "Java 1.8, Tomcat 8, MySQL 5.7"
+    ],
+    "components": [
```

Comment:
One of the suggestions that I made to Stevan was that we should probably be more complete on the list of components that we have and their versions. We should list all of the libraries that we are installing into the Ubuntu system - so bash, openssl, etc.  Anything that we are explicitly installing with a RUN command should probably be added here.

We also need the mysql reference, and I think the JDK / maven versions are out of date?